packages/nx/src/utils/command-line-utils.ts

```diff
@@ -314,7 +314,10 @@ function getMergeBase(base: string, head: string = 'HEAD') {
   }
 }
 
-function getFilesUsingBaseAndHead(base: string, head: string): string[] {
+function getFilesUsingBaseAndHead(
+  base: string = 'main',
+  head: string
+): string[] {
   return parseGitOutput(
```

Comment:
Defaulted to main

Reply:
This is not related to your PR please revert this